ETC_en.tsv

```diff
@@ -4850,32 +4850,32 @@ ETC_20150317_004849	Caught more than five!
 ETC_20150317_004850	Monster [
 ETC_20150317_004851	] was provoked!
 ETC_20150317_004852	Success of field training of Phalaxn!
-ETC_20150317_004853	It's already collected information
+ETC_20150317_004853	This information has already been collected
 ETC_20150317_004854	I drew rubbings
 ETC_20150317_004855	Gathered the rubbing.{nl}Return to Smith Epigraphist!
 ETC_20150317_004856	defeated all interfered Infrobirk{nl}Start the rubbing working!
-ETC_20150317_004857	You are too far away from the soul jar.{nl}In order to collect more souls, you have to near and kill them.
+ETC_20150317_004857	You are too far away from the soul jar.{nl}In order to collect more souls, you have to come closer and kill them.
 ETC_20150317_004858	Forest of Thorns
-ETC_20150317_004859	Defeated the Demon glove which interfere with the monument. {nl}Please check the contents of the monument.
-ETC_20150317_004860	Stat has risen 1 point by using Jeubek.
-ETC_20150317_004861	What has happened?
+ETC_20150317_004859	Defeated Devilglove who interfered with the monument. {nl}Please check the contents of the monument.
+ETC_20150317_004860	Stat has risen by 1 point by using the Jeubek.
+ETC_20150317_004861	What happened?
 ETC_20150317_004862	What should we do?
 ETC_20150317_004863	I request the magic support
-ETC_20150317_004864	Removed the summon circle of the demon!
-ETC_20150317_004865	Come on, Please activate the barrier!
+ETC_20150317_004864	Removed the summonning circle of the demon!
+ETC_20150317_004865	Hurry up! Activate the barrier!
 ETC_20150317_004866	Huff and Puff
 ETC_20150317_004867	Collected fresh water!
-ETC_20150317_004868	The request has been already accepted
-ETC_20150317_004869	Today, It is already running request
-ETC_20150317_004870	Accepted the requests
-ETC_20150317_004871	You have exceeded the request to be carry
-ETC_20150317_004872	You have exceeded your accept limit of daily request
-ETC_20150317_004873	Start Request
```

Comment:
You've changed these few strings where you change 'request' to 'quest'. The Korean strings don't mention quests. It actually does say request.